app/src/main/res/layout/screen_enterotp.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.enterotp.EnterOtpScreen xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/window_background_auth"
+  tools:ignore="Overdraw">
+
+  <LinearLayout
+    android:id="@+id/enterotp_logo_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/color_primary"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/spacing_16"
+    android:paddingTop="@dimen/spacing_4">
+
+    <android.support.v7.widget.Toolbar
+      android:id="@+id/enterotp_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:elevation="0dp"
+      app:navigationIcon="@drawable/ic_arrow_back_24dp"
+      tools:ignore="UnusedAttribute" />
+
+    <TextView
+      android:id="@+id/enterotp_check_sms_sent_to"
+      android:layout_width="wrap_content"
+      android:layout_height="0dp"
+      android:layout_gravity="center_horizontal"
+      android:layout_marginTop="@dimen/spacing_4"
+      android:layout_weight="1"
+      android:ellipsize="end"
+      android:maxLines="1"
+      android:text="@string/enterotp_check_sms_sent_to"
+      android:textColor="@color/white_opacity_70"
```

Comment:
+1 for using translucent text colors. Changing the brand colors will be easy this way.